src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -38,16 +38,9 @@
             <argument type="collection" />
         </service>
 
-        <!--
-            If you want to change the Request class, modify the code in
-            your front controller (app.php) so that it passes an instance of
-            YourRequestClass to the Kernel.
-            This service definition only defines the scope of the request.
-            It is used to check references scope.
-
-            This service is deprecated, you should use the request_stack service instead.
-        -->
-        <service id="request" scope="request" synthetic="true" synchronized="true" />
+        <service id="request" scope="request" synthetic="true" synchronized="true">
```

Comment:
We have a special case in the DI component to trigger a deprecation warning when getting the ``request`` service. It should be removed now that we have a generic feature to deprecate services